lib/src/builder/dart_style_compat.dart

```diff
@@ -0,0 +1,33 @@
+import 'dart:mirrors';
```

Comment:
FYI this isn't the first usage of `dart:mirrors` in over_react's builder; it's also used by `instantiateAnnotation` [here in transformer_utils](https://github.com/Workiva/dart_transformer_utils/blob/cc8e256af7770e3a8d36b3571597b377a34243d5/lib/src/analyzer_helpers.dart#L151-L156), which is used in most of the places the builder reads fields on annotations (e.g., `@Props(...)`, `@Accessor(...)`).